### PR TITLE
senior tribunal caseworker role is caseworker-ia-srcaseofficer

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3878

### Change description ###

Senior caseworker role is caseworker-ia-srcaseofficer

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by leepavlouhmcts — DISMISSED** [review]

File: test/e2e/features/pageObjects/headerPage.js [hunk 1]
```diff
@@ -93,7 +95,13 @@ function HeaderPage() {
 
 
   this.getTabElementWithText = function (tabText) {
-    return this.primaryNavBar.element(by.xpath('//a[contains(text(),"' + tabText + '")]'));
+    return this.primaryNavBar.element(by.xpath('//a[contains(text(),"' + tabText + '")]'));;
```
Comment (connorpgpmcelroy) [inline test/e2e/features/pageObjects/headerPage.js:98]: Change shown here because of two semi-colons - should get rid of one really

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Should remove one of the semi-colons in headerPage.js getTabElementWithText

**Review by leepavlouhmcts — APPROVED** [review]

File: test/e2e/features/pageObjects/headerPage.js (hunk 1 above)
Comment (udaydenduluri33) [reply test/e2e/features/pageObjects/headerPage.js:98]: @sreekanthpuligadda  can you do this as a separate change. This is not a blocker.

**Review by connorpgpmcelroy — APPROVED** [review]
Approving on basis that change mentioned is done in future

**Pull request closed without merging**